src/epics/mutate.js

```diff
@@ -104,8 +47,14 @@ https://github.com/meetup/meetup-web-platform/blob/master/docs/Queries.md#recipe
 				);
 			}
 		})
-		.map(({ payload }) => payload)
-		.mergeMap(doFetch$(getMethodQueryFetch(method, fetchQueries, store)));
+		.map(({ payload: { query, onSuccess, onError } }) => ({
```

Comment:
this is the core of the refactor - instead of making the `fetch` call and parsing the results directly in this middleware, we leave `sync` to make the calls. There is a corresponding update in `sync` to call the custom `onSuccess`/`onError` handlers at the right time.